<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="main_gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="central_gridLayout" columnstretch="1,0,0,0">
     <item row="2" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>50</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="trial_types_label">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Trial name: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="trial_types_comboBox"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="add_trial_label">
       <property name="styleSheet">
        <string notr="true">font: 22pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Add Trial</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QDialogButtonBox" name="ok_btn">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3" rowspan="4">
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_btn</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok_btn</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
